lambdas/src/services/session-request-validator.ts

```diff
@@ -59,6 +64,24 @@ export class SessionRequestValidator {
 
         return payload;
     }
+    private validateStrengthScore(configuredStrengthScore: number, evidenceRequested: EvidenceRequest) {
```

Comment:
[nit] I would expect a function called `validateStrengthScore` to do some sort of validation action rather than just return true if not valid. Maybe we could rename to `isInvalid`? Or better yet, `isValid` and with `else if(!isValid())`

Reply:
Same comment applies for `validateVerificationScore`.